common/src/main/scala/uk/ac/wellcome/models/License.scala

```diff
@@ -0,0 +1,11 @@
+package uk.ac.wellcome.models
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+case class License(
+  val licenseType: String,
+  val label: String,
+  val url: String
+) {
+  @JsonProperty("type") val ldType: String = "License"
```

Comment:
I made up `ldType` (LD = Linked Data) when I started doing these models, but let’s standardise on `ontologyType` instead. I’ll fix our other models to match.